config/image-resize.py

```diff
@@ -0,0 +1,54 @@
+from PIL import Image
+import base64
+from io import BytesIO
+import logging
+
+
+def resize_image(base64_image_str, size, maintain_aspect_ratio=False):
+    """
+    Resize a base64 encoded image to the specified size.
+
+    :param base64_image_str: Base64 encoded image string
+    :param size: Tuple (width, height) specifying the new size
+    :return: resized image or original image object in case of an error
+    """
+    try:
+        # Decode base64 string to image
+        image_data = base64.b64decode(base64_image_str)
+        image = Image.open(BytesIO(image_data))
+
+        width, height = image.size
+        format = image.format
+        logging.info(f'Original size - width: {width}, height: {height}')
+        logging.info(f'Original format: {format}')
+
+        # Check if the image is a GIF and convert to PNG
+        if image.format == "GIF":
+            logging.info("Image is a GIF. Converting to PNG.")
+            image = image.convert("RGBA")
+
+        if maintain_aspect_ratio:
```

Comment:
Will this resize to be *larger* if the size parameter is bigger than the input graphic? For our purposes, this would almost always be undesirable, since it wouldn't add any additional content for ML to chew on, but potentially consume more space/time. Suggest either making this only resize smaller, or if we think someone might have a valid use case for making it larger, make it an additional parameter that can be set.